fe2-android/README.md

```diff
@@ -1,77 +1,39 @@
 # student21_pop: fe2-android
-Proof-of-personhood, spring 2021: Android native front-end
+Proof-of-personhood, Fall 2021: Android native front-end
 
 ## Table of contents
-* [Technicalities](#technicalities)
-* [Setup](#setup)
-* [Github Actions](#github-actions)
-* [Coding Standards](#coding-standards)
+* [Overview](#overview)
+* [Documentation](#documentation)
 
-### Note
+## Overview
 
-This repository is being refactored and some classes and interfaces marked with `@Deprecated` annotation are due for removal.
+#### Setup
 
-## Technicalities
-* The target API is 29 and the minimum required is 26
-* The [R class in Android](https://stackoverflow.com/questions/4953077/what-is-the-class-r-in-android) is an auto-generated class containing the resource IDs all the resources of res/directory
-* The application follows the MVVM pattern, and uses this [guide to app architecture](https://developer.android.com/jetpack/guide)
-* The [ViewModel](https://developer.android.com/topic/libraries/architecture/viewmodel) classes implement the UI logic, retrieve and manage the data that is needed
-* The WebSockets are implemented using the [Scarlet](https://github.com/Tinder/Scarlet) library, one of its usage on this application is in the [Injection](https://github.com/dedis/student_21_pop/blob/master/fe2-android/app/src/prod/java/com/github/dedis/student20_pop/Injection.java) class
-* The application uses [RxJava](https://github.com/ReactiveX/RxJava) for asynchronous and event based programs
```

Comment:
I assume these parts will be moved later on?

Reply:
in docs/README.md